virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc

```diff
@@ -0,0 +1,19 @@
+[id="virt-configuring-a-watchdog"]
+= Configuring a watchdog
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-a-watchdog
+
+toc::[]
+
+Expose a watchdog by configuring the virtual machine (VM) for a watchdog device, installing the watchdog, and starting the watchdog service.
+
+== Prerequisites
+
+* Kernel support for an `i6300esb` watchdog device.
+
+include::modules/virt-defining-a-watchdog.adoc[leveloffset=+1]
+include::modules/virt-installing-a-watchdog.adoc[leveloffset=+1]
+
+.Additional information
```

Comment:
In an assembly, the add'l info section should start with `==` (the `.Additional information` construction is for modules). Any heading needs an anchor ID. You can update this to:

```
[id="{context}_additional-resources"]
== Additional resources
```

The prereq section correctly uses `==`, but also needs an anchor ID. You can update it to:

```
[id="{context}_prerequisites"]
== Prerequisites
```

Most of the files in the repo manually write out the title instead of using `{context}` here, but I haven't had a problem with it. It just needs to be a unique ID.

More info about this: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
Updated, and I'll take a look at that resource. Current order of modules is per dev suggestion in the depths of these edits.